packages/theme-toggler-extension/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "jupyterlab-theme-toggler-extension",
```

Comment:
Similar comment as above, maybe we should stick to `jupyterlab-theme-toggle` for now?

https://www.npmjs.com/package/jupyterlab-theme-toggle